pystac/summaries.py

```diff
@@ -194,16 +202,21 @@ def summarize(
 
 
 class Summaries:
+    lists: Dict[str, List[Any]]
+    other: Dict[str, Any]
+    ranges: Dict[str, RangeSummary[Any]]
+    schemas: Dict[str, Dict[str, Any]]
+
     def __init__(
         self, summaries: Dict[str, Any], maxcount: int = DEFAULT_MAXCOUNT
     ) -> None:
         self._summaries = summaries
         self.maxcount = maxcount
```

Comment:
These should go in the class properties, yes?

Reply:
I'm not sure I follow what you are suggesting? Could you maybe provide a suggested change or a code snippet to illustrate what you are thinking?

Reply:
@schwehr Do you mean like `lists: Dict[str, List[Any]]` above? If so, I don't think it's necessary. Mypy knows the type of `maxcount`, so it will infer the type of `self.maxcount`. It can't do the same for empty lists and dicts, which is why they need to be declared at the class level.

Reply:
Think of the humans 😀.  Also, just because there is a type in the init, doesn't mean the type never changes in the class attribute.  The reader shouldn't assume

Reply:
Okay, I see now. Yes, @schwehr my intention here was to have any instance attributes be typed at the class level. As you said, even if it's not strictly required by `mypy`, I do think it's helpful as documentation.

Reply:
Fixed in 152d951918a3952d868ee208f03f86da60763ceb